ods_tools/oed/exposure.py

```diff
@@ -229,3 +244,96 @@ def check(self, validation_config=None):
             validation_config = self.validation_config
         validator = Validator(self)
         return validator(validation_config)
+
+    def to_version(self, version):
+        """
+        goes through location to convert columns to specific version.
+        Right now it works for OccupancyCode and ConstructionCode.
+        (please note that it only supports minor version changes in "major.minor" format, e.g. 3.2, 7.4, etc.)
+
+        Args:
+            version (str): specific version to roll to
+
+        Returns:
+            itself (OedExposure): updated object
+        """
+
+        def version_to_tuple(version_str):
+            """Converts the version in string format to a tuple of integers"""
+            return tuple(map(int, version_str.split('.')))
+
+        def strip_version(version_str):
+            """
+            Strips a version string to the format 'major.minor'.
+
+            Args:
+                version_str (str): The version string.
+
+            Returns:
+                str: The stripped version string.
+            """
+            # Use regex to find 'major.minor' format
+            match = re.match(r"(\d+\.\d+)", version_str)
+            if match:
+                return match.group(1)
+            else:
+                raise ValueError(f"Invalid version format: {version_str}")
+
+        # Strip the version string down to 'major.minor' format
+        stripped_version = strip_version(version)
+
+        # Input checks - format, version, and convert to tuple
+        if not re.match(r"^\d+\.\d+$", stripped_version):
+            raise ValueError(
+                "Version should be provided in 'major.minor' format, e.g. 3.2, 7.4, etc."
+            )
+
+        if stripped_version not in self.oed_schema.schema["versioning"]:
```

Comment:
I don't think we want that you could have a version with no change of the code, specially with the versioning we use to have.
You can set a minimum supported version but not ask for all the intermediary version to be present